cfgov/cfgov/urls.py

```diff
@@ -409,7 +410,18 @@ def this_view_always_raises_http404(request, *args, **kwargs):
         'FINANCIAL_WELLBEING_HUB',
         r'^practitioner-resources/financial-well-being-resources/',
         'financial-well-being-resources'
-    )
+    ),
+
+    # Temporary: HMDA Legacy pages
+    # Will be deleted when HMDA API is retired (hopefully Summer 2019)
+    flagged_url('HMDA_LEGACY_REVIEW',
```

Comment:
FWIW, there exists an (undocumented) `flagged_wagtail_template_view` helper that would allow you to eliminate the hmda/views.py file and just refer to the template directly in urls.py. That would look like this:

```py
urlpatterns = [
    ...
    url(r'data-research/hmda/explore$',
        flagged_wagtail_template_view(
            'HMDA_LEGACY_REVIEW',
            'orange-explorer.html'
        ),
        name='legacy_explorer_draft'
    ),
    ...
]
```

I'm not sure this is an improvement over what is already here, which maybe indicates that the helper could be made better somehow.